### PR TITLE
v1.18.2

### PR DESCRIPTION
**Pull request by nim65s**
Hi,

This is an update following https://github.com/ros/rosdistro/blob/4fbed76b297a148cd5ec7f7f3a882a28e02005c6/noetic/distribution.yaml#L1022

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -4,7 +4,7 @@ pkgdesc="ROS - The control toolbox contains modules that are useful across all c
 url='https://wiki.ros.org/control_toolbox'
 
 pkgname='ros-noetic-control-toolbox'
-pkgver='1.17.0'
+pkgver='1.18.2'
 _pkgver_patch=0
 arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
 pkgrel=2
```
Comment (bionade24) [inline PKGBUILD]: Wrong, `pkgrel` needs to be reset to 1

**Review by bionade24 — CHANGES_REQUESTED** [review]
Please reset `pkgrel` after updates. You can use https://github.com/ros-arch/ros-aur-helpers to update the PKGBUILDs automatically.

**Comment by bionade24** [comment]
Please use `git commit --amend` instead of making a new commit, otherwise the AUR will reject it becase there isn't one version increase per commit. If we otherwise forget to squash merge it will be fucked up.

**Comment by nim65s** [comment]
Thanks for the review and the tips !

**Review by bionade24 — APPROVED** [review]

**Review by acxz — APPROVED** [review]

**Pull request merged**